client/js/components/main/assessment.spec.js

```diff
@@ -38,7 +37,8 @@ describe('assessment', function() {
     };
 
     progress = {
-      currentItemIndex:0
+      currentItemIndex:0,
+      answerMessageIndex:[]
```

Comment:
Why is an "index" being assigned an array, instead of an integer, `null`, or `undefined`?